projects/storefrontapp-e2e-cypress/cypress/integration/regression/checkout/checkout-flow.core-e2e-spec.ts

```diff
@@ -25,5 +31,72 @@ context('Checkout flow', () => {
       checkout.placeOrderWithCheapProduct(user);
       checkout.verifyOrderConfirmationPageWithCheapProduct(user);
     });
+
+    it('should filter with faceting and perform checkout', () => {
+      const user = getSampleUser();
+      checkout.visitHomePage();
+
+      checkout.clickHamburger();
+
+      checkout.registerUser(false, user);
+      searchResult();
+      filterUsingFacetFiltering();
+
+      cy.intercept(
```

Comment:
Looks good now. The only thing I would say is, maybe extract all these into a function in checkout-flow.ts file. Can call it anything, maybe checkoutUsingFacetting() or checkoutFirstDisplayedProduct().
So we keep this spec test file more user readable and short.